database/index.js

```diff
@@ -262,27 +264,28 @@ var toExport = {
                 });
             });
         },
+
         getPlaceMeasurements: function(ids, types, start, end){
             return databaseP.then(function(db){
 
                 // if no dates provided, assume we want all
-                var start = start ? start : new Date("1900-10-15T11:23:19.766Z");
-                var end = end ? end : new Date("2200-10-15T11:23:19.766Z");
+                var _start = start || new Date('1900-10-15T11:23:19.766Z');
+                var _end = end || new Date('2200-10-15T11:23:19.766Z');
```

Comment:
tricky error I did corrected here by @4rzael : when you write var a, a is undefined and thus the ? test alwas falls back on the second choice